app/webpacker/packs/application.js

```diff
@@ -1,6 +1,9 @@
 require.context('govuk-frontend/govuk/assets');
 
 import '../styles/application.scss';
+
+const images = require.context('../images', true)
```

Comment:
I'm not sure I understand why this is set as a `const` - could you add some context?

Reply:
Fair enough.
I cut and pasted and did not come back to tidy it up.
Gone soon

Reply:
About the assets that are not used, I deliberately copied them all because I expected they would be used in some part of the prototype code I have not seen yet.  And because they are tiny, 196 bytes

The print icons are needed, at least one